plugins/modules/events.py

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import, division, print_function
+
+import os
+import requests
+
+
+__metaclass__ = type
+
+DOCUMENTATION = r"""
+---
+module: events
+
+short_description: Handle AssistedInstall cluster events
+
+version_added: "1.0.0"
+
+description: Interact with cluster events through the AssistedInstall API
+
+options:
+    action:
+        description: The action to perform.
+        required: false
+        default: "list"
+    cluster_id:
+        description: The cluster ID to perform the action on.
+        required: false
+    limit:
+        description: The maximum number of records/events to retrieve.
+        default: 10
```

Comment:
@agopala95 Where is the default value of 10 defined for the limit?

Reply:
I just left a placeholder value that could be modified later on but I guess I can remove this for now.

Reply:
Removed

Reply:
Removed